advanced/configuring-comments-callbacks.md

```diff
@@ -1,40 +1,52 @@
 ---
 layout: default
-title: Configuring callbacks for the Comments plugin
-title_nav: Configuring callbacks for the Comments plugin
-description: Instructions for configuring callbacks for the Comments plugin
+title: Configuring the Comments plugin in callback mode
+title_nav: Callback mode
+description: Information on configuring the Comments plugin in callback mode
 keywords: comments commenting tinycomments callback
 ---
 
-**Callback mode** is the default mode for [the Comments plugin]({{site.baseurl}}/plugins/premium/comments/). In the callback mode, the user needs to configure storage to be able to save comments on the server. The Comments functions (create, reply, edit, delete comment, delete all conversations, and lookup) are configured differently depending upon the server-side storage configuration.
+{% assign pluginname = "Comments" %}
+{% assign plugincode = "comments" %}
 
-### Required settings
+**Callback mode** is the default mode for [the Comments plugin]({{site.baseurl}}/plugins/premium/comments/). In the callback mode, callback functions are required to save user comments on a server. The Comments functions (create, reply, edit, delete comment, delete all conversations, resolve, and lookup) are configured differently depending upon the server-side storage configuration.
```

Comment:
Why do we have a page with duplicate content? Should this instead be pulling the duplicate content via an `include`?

Honestly, since this page has been incorporated into the core plugin docs it feels as though this page should be deleted instead and a redirect setup.

Reply:
A redirect is being set up, however, this PR will be released before the redirect is. I'll be deleting this page after the redirect goes live.